src/lambda/LambdaFunction.js

```diff
@@ -136,6 +143,7 @@ export default class LambdaFunction {
 
     // TEMP
     const funOptions = {
+      architecture: this.#architecture,
       codeDir: this.#codeDir,
       functionKey,
       functionName: name,
```

Comment:
This field was deleted from DockerContainer class and is not used in HandlerRunner.